lib/iris/tests/integration/test_FieldsFileVariant.py

```diff
@@ -24,12 +24,22 @@
 
 import shutil
 import tempfile
+import unittest
 
 import numpy as np
 
 from iris.experimental.um import (Field, Field2, Field3, FieldsFileVariant,
                                   FixedLengthHeader)
 
+try:
+    import mo_unpack
```

Comment:
I'm thinking this is wrong, since this causes tests to be skipped in the Travis-CI build...